dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/internal/commands/PackageProcessor.java

```diff
@@ -788,4 +797,39 @@ private boolean java2SecurityEnabled() {
     private boolean isArchiveJar() {
         return packageFile.getName().endsWith(".jar");
     }
+
+    // Reads the java9.options file
+    private HashMap<String, String> readJava9Options() throws IOException {
+        HashMap<String, String> hm = new HashMap<String, String>();
+        StringBuffer exports = new StringBuffer();
+        StringBuffer opens = new StringBuffer();
+        BufferedReader r = new BufferedReader(new FileReader(installRoot.getAbsolutePath() + File.separator + "lib" + File.separator + "platform" + File.separator + "java"
```

Comment:
Perhaps we should make this a bit more tolerant and just log a warning if the java9.options cannot be found/read? Would be bad if server start blew up on something that was just trying to avoid a warning message.

Reply:
I decided to leave as-is.  It will result in: 

CWWKE0056E: Unable to zip up directory due to an IO exception java.io.FileNotFoundException: ${wlp.install.root}\lib\platform\java\java9.options (The system cannot find the file specified).
Server test1 package failed. Check server logs for details.

Which in my mind tells that their install is not correct when running java 9.